website/docs/cloud-docs/api-docs/stacks.mdx

```diff
@@ -0,0 +1,946 @@
+        "service-provider": "github",
+        "trigger-disabled": false
+      },
+      "created-at": "2025-05-09T16:40:31.255Z",
+      "updated-at": "2025-05-09T20:24:47.324Z"
+    },
+    "relationships": {
+      "project": {
+        "data": {
+          "id": "prj-18PG93QyqCMhLhWT",
+          "type": "projects"
+        }
+      }
+    },
+    "links": {
+      "self": "/api/v2/stacks/st-EJyw2MktbRjdsg96",
+      "stack-configurations": "/api/v2/stacks/st-EJyw2MktbRjdsg96/stack-configurations",
+      "stack-deployments": "/api/v2/stacks/st-EJyw2MktbRjdsg96/stack-deployments"
+    }
+  }
+}
+```
+
+## Delete a Stack
+
+`DELETE /stacks/:policy_id`
+
+This endpoint deletes the specified Stack.
+
+| Parameter   | Description                                                                                                                                           |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `:stack_id` | The ID of the stack to delete. Refer to [List Stacks](/terraform/cloud-docs/api-docs/stacks#list-stacks) for reference information about finding IDs. |
+
+| Status  | Response                  | Reason                                                  |
+|---------|---------------------------|---------------------------------------------------------|
+| [204][] | No Content                | Successfully deleted the stack                          |
+| [404][] | [JSON API error object][] | Stack not found, or user unauthorized to perform action |
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE \
+  https://app.terraform.io/api/v2/stacks/st-EJyw2MktbRjdsg96
+```
+## Create a stack configuration
+
+This endpoint creates a stack configuration for the specified stack.
+
+`POST /stacks/:stack_id/stack-configurations`
+
+| Parameter   | Description                                   |
+|-------------|-----------------------------------------------|
+| `:stack_id` | The stack id to create the configuration for. |
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+| Key path                    | Type    | Default | Description                                                                                                                                    |
+|-----------------------------|---------|---------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `data.attributes.plan-only` | boolean | `false` | Specifies if this is a [speculative](/terraform/cloud-docs/run/modes-and-options#plan-only-speculative-plan) plan that Terraform cannot apply. |
```

Comment:
I think we can remove the link here and change the verbiage in the description to something like `Specifies if this is a speculative configuration that HCP Terraform cannot converge past the plan step` 🤷